src/pages/resource/EmbedButtons.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { FC } from 'react';
+import { Button, Grid, Typography } from '@material-ui/core';
+import styled from 'styled-components';
+import { Resource } from '../../types/resource.types';
+import { LMSTypes } from '../../types/lms.types';
+import { useTranslation } from 'react-i18next';
+import { embed } from '../../utils/lmsService';
+import { LMSParametersName } from '../../types/LMSParameters';
+
+const StyledGrid = styled(Grid)`
+  margin-top: 2rem;
+`;
+
+const StyledButtons = styled(Button)`
+  margin-right: 1rem;
+  margin-top: 1rem;
+`;
+
+const small = {
+  horizontal: 560,
+  vertical: 315,
+};
+
+const medium = {
+  horizontal: 640,
+  vertical: 360,
+};
+
+const large = {
+  horizontal: 853,
+  vertical: 480,
+};
+
+interface EmbedButtonsProps {
+  resource: Resource;
+}
+
+const EmbedButtons: FC<EmbedButtonsProps> = ({ resource }) => {
+  const { t } = useTranslation();
+  const searchParams = new URLSearchParams(window.location.search);
```

Comment:
Vil ikke all parameter-håndteringa bli kjørt for ofte ? Lønner det seg med useeffekt eller slipper vi det med windows.location ?

Reply:
fikset